streaming/src/main/scala/org/apache/carbondata/streaming/StreamSinkFactory.scala

```diff
@@ -77,6 +78,22 @@ object StreamSinkFactory {
     }
   }
 
+  lazy val listenerAdded = new mutable.HashMap[Int, Boolean]()
+
+  // add CarbonStreamingQueryListener to other SparkSession
```

Comment:
please change to `add CarbonStreamingQueryListener if it is SparkSession instance`